client/src/components/Navbar/Menus/Authenticated/Mobile.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { makeStyles, Drawer, Button } from '@material-ui/core';
+import { GiHamburgerMenu } from 'react-icons/gi';
+import List from './List';
+
+const useStyles = makeStyles((theme) => ({
+  list: {
+    minWidth: 350,
+    width: 'auto',
+  },
+  burger: {
+    fontSize: 26,
+    [theme.breakpoints.down('md')]: {
+      fontSize: 22,
+    },
+  },
+}));
+
+export const Mobile = (): JSX.Element => {
+  const classes = useStyles();
+  const [isOpen, setIsOpen] = React.useState<boolean>(false);
```

Comment:
just a personal style thing, but can we just `import React, { useState }  from 'react';` so that we don't have `React.useState`

Reply:
Will that be our preference across the project? I've personally enjoyed `React.useSomeHook` as that way I don't cloud up imports but that works well for me too!